includes/Core/Util/Activation.php

```diff
@@ -155,9 +155,81 @@ private function get_activation_notice() {
 					ob_start();
 					?>
 					<script type="text/javascript">
-					if( 'undefined' !== typeof sendAnalyticsTrackingEvent ) {
-						sendAnalyticsTrackingEvent( 'plugin_setup', 'plugin_activated' );
-					}
+						if( 'undefined' !== typeof sendAnalyticsTrackingEvent ) {
+							sendAnalyticsTrackingEvent( 'plugin_setup', 'plugin_activated' );
+						}
+
+						jQuery(document).ready(function() {
+							var trackingScriptPresent = !! googlesitekit.admin.trackingOptIn;
+
+							if ( googlesitekit.admin.trackingOptIn ) {
+								jQuery('#opt-in').attr( 'checked', googlesitekit.admin.trackingOptIn );
+							}
+							if ( googlesitekit.admin.proxySetupURL ) {
+								jQuery( '#start-setup-link' ).attr( 'href', googlesitekit.admin.proxySetupURL );
+							}
+
+							// TODO: Fix this; it doesn't currently work as
+							// `sendAnalyticsTrackingEvent` is not synchronous
+							// but fails to implement an async API that confirms
+							// the event was sent. This was moved from an
+							// `onClick` handler, but did not work there either.
```

Comment:
Are we sure about this? Obviously the response is not important (or likely sent at all) here and they have been collecting these events so it has been working at least some/most of the time. Would be nice to know if there were cases where it wasn't working though.

Reply:
Oh, right, maybe not. But it's a bit spotty. At any rate I think we need to check out the `beacon` attribute. @adamsilverstein mentioned it in a Slack chat but I forgot about it once I revisited this PR. I'll take a look at that and remove this `TODO`. 👍